### PR TITLE
community meeting poll blog post

### PR DESCRIPTION
**Pull request by stockholmux**
*Description of changes:* Added a blog post about our upcoming community meeting and the active poll

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: _posts/2020-12-18-community-meeting-poll.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+layout: posts
+author: Kyle Davis
+comments: true
+title: "2021 Open Distro for Elasticsearch Community Meetings"
+categories:
+- meetings
+feature_image: "https://d2908q01vomqb2.cloudfront.net/ca3512f4dfa95a03169c5a670a4c91a19b3077b4/2019/03/26/open_disto-elasticsearch-logo-800x400.jpg"
+---
+
+We've been having regular bi-weekly community meetings over Zoom at 10 am Pacific on Mondays for quite some time.  While there has certainly been good response and attendance, we recognize that this time could be less than ideal for those who live in timezones with a UTC greater than +4 (India and points west to the international dateline).
```
Comment (aetter) [inline _posts/2020-12-18-community-meeting-poll.md:11]: Recommend "10:00 AM Pacific" (or PT if you prefer)

Comment (aetter) [reply _posts/2020-12-18-community-meeting-poll.md:11]: I'd try to be more specific and call out our broad userbase here. Something like, "we recognize that this time isn't ideal for our community members in India, Russia, Japan, South Korea, Australia, and everyone else located closer to the west side of the International Date Line."

File: _posts/2020-12-18-community-meeting-poll.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Now through December 31, 2020, there is a poll over on [the Open Distro forums](https://discuss.opendistrocommunity.dev/) about times/days for these meetings. You can take part in this survey by [logging into the forum and voting on a day and time](https://discuss.opendistrocommunity.dev/t/2021-open-distro-community-meetings/4404). It only takes a second and it will really help us understand when we can reach the most community members.
```
Comment (aetter) [inline _posts/2020-12-18-community-meeting-poll.md:13]: about times and days for these meetings.

It only takes a few seconds and will really help us understand how to reach the broadest range of community members.

File: _posts/2020-12-18-community-meeting-poll.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Several possibilities are on the table:
+
+* keep it the same day of the week/time,
```
Comment (aetter) [inline _posts/2020-12-18-community-meeting-poll.md:17]: Capitalize all of these and period after.

* Keep the time and day the same.
* Adopt a totally new time and day.
* Rotate times and days of the week.
* Have multiple meetings for different parts of the world.

File: _posts/2020-12-18-community-meeting-poll.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* adopt a totally new day of the week and/or time,
+* rotate times and/or days of the week,
+* have multiple meetings for different parts of the world.
+
+If you have ideas how to best handle the meeting scheduling, [please vote and comment on our thread](https://discuss.opendistrocommunity.dev/t/2021-open-distro-community-meetings/4404). We welcome input on this important pillar of our community.
```
Comment (aetter) [inline _posts/2020-12-18-community-meeting-poll.md:22]: I don't love the second sentence---just a bit impersonal. Maybe something like, "Your input here keeps the project inclusive and helps us better understand the goals and needs of our wide-ranging community." Tweak/discard/etc.

File: _posts/2020-12-18-community-meeting-poll.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Note: Due to the holidays and planned vacation, we **will not have a meeting on our normal cadence (Dec 28, 2020)**. Meetings will resume in early in the new year, hopefully with a newly adopted schedule.
```
Comment (aetter) [inline _posts/2020-12-18-community-meeting-poll.md:24]: December 28, 2020

the new year, hopefully with a new schedule.

File: _posts/2020-12-18-community-meeting-poll.md (hunk 4 above)
Comment (stockholmux) [reply _posts/2020-12-18-community-meeting-poll.md:22]: Yours take is way better!

**Pull request merged**